source/reference/minio-server/settings/metrics-and-logging.rst

```diff
@@ -0,0 +1,865 @@
+.. _minio-server-envvar-metrics-logging:
+
+======================================================
+Environment Variables to Configure Metrics and Logging
```

Comment:
We have configuration settings on this page too.

"Metrics and Logging Settings"